README.md

```diff
@@ -18,48 +18,125 @@ The API is documented in the [docs/api](./docs/api) directory of this repository
 
 ## Quickstart
 
-The backend uses `OpenShift`'s built-in `OAuth` Server to authenticate users. To run it, configure environment variables in a new `.env` file:
+### Local Development
+
+To run the backend locally, a `.env` file is used where several environment variables are set.
 
 ```bash
-# Define the allowed origin regex pattern for WebSocket connections from the frontend. If not specified, all origins will be approved.
-ALLOWED_ORIGIN_REGEX="http:localhost:8080|https:example.com.*"
-
-# Cluster configuration
-CLUSTER_NAME=ocp-rcs-example    # Change to match your own
-CLUSTER_DOMAIN=dana.com         # Change to match your own
-KUBE_AUTH_BASE_URL="https://oauth-openshift.apps.${CLUSTER_NAME}.${CLUSTER_DOMAIN}"
-KUBE_API_BASE_URL="https://api.${CLUSTER_NAME}.${CLUSTER_DOMAIN}:6443"
-
-INSECURE_SKIP_VERIFY=true
-KUBE_CLIENT_ID="openshift-challenging-client"
-KUBE_AUTH_URL="${KUBE_AUTH_BASE_URL}/oauth/authorize"
-KUBE_TOKEN_URL="${KUBE_AUTH_BASE_URL}/oauth/token"
-KUBE_USERINFO_URL="${KUBE_API_BASE_URL}/apis/user.openshift.io/v1/users/~"
-KUBE_API_SERVER="${KUBE_API_BASE_URL}"
-
-cat <<EOF > .env
-INSECURE_SKIP_VERIFY=${INSECURE_SKIP_VERIFY}
-KUBE_CLIENT_ID=${KUBE_CLIENT_ID}
-KUBE_AUTH_URL=${KUBE_AUTH_URL}
-KUBE_TOKEN_URL=${KUBE_TOKEN_URL}
-KUBE_USERINFO_URL=${KUBE_USERINFO_URL}
-KUBE_API_SERVER=${KUBE_API_SERVER}
-ALLOWED_ORIGIN_REGEX=${ALLOWED_ORIGIN_REGEX}
-EOF
+$ make env-file CLUSTER_NAME=<CLUSTER_NAME> CLUSTER_DOMAIN=<CLUSTER_DOMAIN>
+```
+
+For example:
+
+```bash
+$ make env-file CLUSTER_NAME=ocp-test CLUSTER_DOMAIN=dana.com
+```
+
+#### Adding a new variable
+
+To create a new environment variable, both locally and for the production deployment to use, create a new entry in the `charts/platform-backend/_config-data.yaml` file. Then, set its value in the `charts/platform-backend/values.yaml` file:
+
+For example:
+
+`config-data.yaml`:
+
+```diff
+INSECURE_SKIP_VERIFY: "{{ .Values.config.insecureSkipVerify }}"
++ NEW_VARIABLE: "{{ .Values.config.newVariable }}"
+```
+
+`values.yaml`:
+
+```diff
+config:
+  # -- Name of the ConfigMap where authentication endpoints are stored
+  name: config
++  # -- Indicates a value
++  newVariable: <VALUE>
 ```
 
+#### Running locally
+
 Then, use the following `Makefile` target in order to run the backend:
 
 ```bash
 $ make run
 ```
 
-Alternatively, to build the backend as a Docker image, use the following `Makefile` targets:
+### Deploy to OpenShift
+
+#### Prereq
+
+To install all the prerequisites to run the `platform-backend` on an OpenShift cluster, use the following `Makefile` target, which configures the `Capp` environment to work properly using an actual DNS Server.
```

Comment:
can you address the minimal amount of value you need for the make prereq to work ?

Reply:
For a fully functioning environment you need all the values listed in the table below.